packages/serializers/serializer/package.json

```diff
@@ -39,9 +39,9 @@
   "peerDependencies": {
     "react": ">=16.8.0",
     "react-dom": ">=16.8.0",
-    "slate": ">=0.60.0",
-    "slate-history": ">=0.60.0",
-    "slate-react": ">=0.60.0"
+    "slate": ">=0.66.0",
```

Comment:
I guess we need that in plate package as well?

Reply:
I was matching what was there previously which was plate had a specific version and the serializers had a range. Let me know if you prefer a range everywhere and I'll update.